sdk/documenttranslation/azure-ai-documenttranslation/README.md

```diff
@@ -1,35 +1,402 @@
+
+Basic information about HTTP sessions (URLs, headers, etc.) is logged at `INFO` level.
+
+Detailed `DEBUG` level logging, including request/response bodies and **unredacted**
+headers, can be enabled on the client or per-operation with the `logging_enable` keyword argument.
+
+See full SDK logging documentation with examples [here][sdk_logging_docs].
+
+### Optional Configuration
+
+Optional keyword arguments can be passed in at the client and per-operation level.
+The azure-core [reference documentation][azure_core_ref_docs]
+describes available configurations for retries, logging, transport protocols, and more.
+
+## Next steps
+
+The following section provides several code snippets illustrating common patterns used in the Document Translation Python client library.
+
+### More sample code
+
+These code samples show common scenario operations with the Azure Document Translation client library.
+
+* Client authentication: TODO
+* Create a translation job: TODO
+* Check the status of documents: TODO
+* List all submitted translation jobs: TODO
+
+
+### Async samples
+This library also includes a complete async API supported on Python 3.6+. To use it, you must
+first install an async transport, such as [aiohttp](https://pypi.org/project/aiohttp/). Async clients
+are found under the `azure.ai.documenttranslation.aio` namespace.
+
+* Client authentication: TODO
+* Create a translation job: TODO
+* Check the status of documents: TODO
+* List all submitted translation jobs: TODO
+
+### Additional documentation
+
+For more extensive documentation on Azure Cognitive Services Document Translation, see the [Document Translation documentation][python-dt-product-docs] on docs.microsoft.com.
+
+## Contributing
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit [cla.microsoft.com][cla].
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][code_of_conduct]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
+
+<!-- LINKS -->
+
+[python-dt-src]: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation
+[python-dt-pypi]: https://aka.ms/azsdk/python/texttranslation/pypi
+[python-dt-product-docs]: https://docs.microsoft.com/azure/cognitive-services/translator/document-translation/overview
+[python-dt-ref-docs]: https://aka.ms/azsdk/python/documenttranslation/docs
```

Comment:
i think this one should be "https://docs.microsoft.com/en-us/rest/api/cognitiveservices/translator/documenttranslation"

Reply:
Good catch, I updated the aka link to point to the service docs. Once we release the SDK, we can re-point it to our documentation.